### PR TITLE
20200402 Feilong meeting notes

### PR DESCRIPTION
**Pull request by mfriesenegger**
Signed-off-by: Mike Friesenegger <mikef@suse.com>

### CODE REVIEW

File: feilong_meetings/20200402-meeting_notes.md [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+- Ji Chen
+- Daniel Pono Takamori
+- James Vincent
+- Johan Schelling
+- Len Santalucia
+- Nick Snel
+- Vinnie Terrone
+- John Mertic
+- Alex Dumitru
+- Rudy Grigar
```
Comment (bramwelt) [inline feilong_meetings/20200402-meeting_notes.md:15]: ` - Trevor Bramwell`
I didn't say much but I was here. :wave:

Comment (mfriesenegger) [reply feilong_meetings/20200402-meeting_notes.md:15]: I will add your name and amend the PR.

Comment (mfriesenegger) [reply feilong_meetings/20200402-meeting_notes.md:15]: Sorry that I missed you.  Your name is added.

**Comment by jichenjc** [comment]
thanks @mfriesenegger ! :)

**Pull request merged**